app/js/user.js

```diff
@@ -88,6 +87,49 @@
           });
         });
       });
+    },
+
+    /**
+     * Updates the local user data as well as last sync time.
+     * @param {object} user_data - The ammended user data and settings i.e.
+     * user_data = {
+     *   user: user,
+     *   settings, settings
+     * };
+     * @returns The updated user object from indexedDB
+     */
+    update_user: function(user_data) {
+      SumoDB.update_user(user_data).then(function(user) {
+        user.last_sync = Date.now();
+        return asyncStorage.setItem('user', user).then(function() {
```

Comment:
Added a comment here https://bugzilla.mozilla.org/show_bug.cgi?id=1074947#c10 - we could potentially use settings to store this in along with new_comment_notification and buddyup_reminder.

If that makes sense to you, I can actually just go ahead and do that. Thoughts @Rik ?